### PR TITLE
gh actions to test helm charts

### LINKED ISSUE #13: Add install/lint testing for Helm charts
### Name and Version

charts/*

### What is the problem this feature will solve?

Automatic testing to prevent breakages and enforce best standards.

### What is the feature you are proposing to solve the problem?

A Github Action to run lint and install tests for every PR/commit pushed to OSDFIR Infrastructure.

Reference here: https://helm.sh/docs/helm/helm_lint/ and https://github.com/helm/chart-testing

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by wajihyassine**
### Description of the change

Adds github actions helm chart testing via https://github.com/helm/chart-testing. Lint tests then installs the helm charts helping ensure that the chart is functioning properly

### Applicable issues

- fixes #13

### CODE REVIEW

**Comment by wajihyassine** [comment]
Excluding OSDFIR Infra chart for now until we merge the charts into a single one. 

This is due to the osdfir infra chart needing the updated TS/Turbinia changes after the Turbinia/TS changes have been been pushed to the registry

**Comment by wajihyassine** [comment]
@hacktobeer if you can help with a full review and @aarontp / @berggren high level awareness review

**Review by berggren — APPROVED** [review]

File: .github/ct.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+remote: origin
+target-branch: main
+chart-repos:
+  - bitnami=https://charts.bitnami.com/bitnami
+  - opensearch=https://opensearch-project.github.io/helm-charts
+chart-dirs:
+  - charts
+check-version-increment: false
```
Comment (hacktobeer) [inline .github/ct.yaml:8]: remembering your comment on one of my PRs...don't you want this enabled?

File: charts/timesketch/values.yaml [hunk 2]
```diff
@@ -309,7 +309,9 @@ opensearch:
     ## requests:
     ##    cpu: 500m
     ##    memory: 1Gi
-    requests: {}
+    requests:
+      cpu: 250m
```
Comment (hacktobeer) [inline charts/timesketch/values.yaml:313]: If the opensearch params are set to 512M would it not make sense to set the pod to the same?

**Review by hacktobeer — APPROVED** [review]
LGTM, small nits

File: charts/timesketch/values.yaml (hunk 2 above)
Comment (wajihyassine) [reply charts/timesketch/values.yaml:313]: Agree I bumped this to 512Mi

File: .github/ct.yaml (hunk 1 above)
Comment (wajihyassine) [reply .github/ct.yaml:9]: Enabled for now and will test how it evals as was a bit iffy with this given we dont have automated releasing yet either and tedious to bump each chart for the values.yaml. Once we move to a single chart instead would be less of an issue

**Pull request merged**